# Target repository: vendor/gms
# Checked against Evolution-X vendor_gms d5490a1323f348fff1cc35f091b29ba583c67a84 (branch cnb, crDroid 17.0).
# Exclude incompatible Pixel package prefixes from the OnePlus product lists, using the
# package names of branch cnb, gms_pico.mk included. AiAi, GoogleRestore and the core
# GMS packages stay.

diff --git a/gms_full.mk b/gms_full.mk
index 850d1b8..9ec0f06 100644
--- a/gms_full.mk
+++ b/gms_full.mk
@@ -111,13 +111,11 @@ PRODUCT_PACKAGES += \
     CustomizationBundlePrebuiltFullVersion \
     DeviceIntelligenceNetworkPrebuiltAstrea \
     DevicePersonalizationAiAiPrebuiltPixel2026 \
-    FamilySpacePrebuilt-v2026 \
     FilesPrebuilt \
     GeminiShell_227 \
     GoogleDialer \
     GoogleRestorePrebuilt-v1054408 \
     KidsSupervisionStub \
-    OdadPrebuilt \
     PartnerSetupPrebuilt \
     Phonesky \
     PrebuiltBugle \
@@ -127,14 +125,11 @@ PRODUCT_PACKAGES += \
     RelationshipsPrebuilt-301 \
     SafetyHubPrebuilt \
     ScribePrebuilt_v8.7.880674799 \
-    SearchSelectorPrebuilt \
     SettingsIntelligenceGooglePrebuilt \
-    SetupWizardPrebuilt_cd1a \
     TurboPrebuilt \
     Velvet \
     VerifierPrebuiltClassic \
     WallpaperEffect \
-    WeatherPixelPrebuilt_10008663 \
     WellbeingPrebuilt
 
 # system/app
@@ -145,7 +140,6 @@ PRODUCT_PACKAGES += \
 # system/priv-app
 PRODUCT_PACKAGES += \
     DocumentsUIGoogle \
-    GooglePackageInstaller \
     TagGoogle
 
 # system_ext/app
@@ -162,16 +156,12 @@ endif
 
 PRODUCT_PACKAGES += \
     AvatarPickerGoogle \
-    DeviceConnectivityServicePrebuilt_26.01.00 \
     GoogleFeedback \
     GoogleServicesFramework \
     MoseyApp \
-    NexusLauncherRelease \
     PixelSearchPrebuilt_26.02.00 \
-    SetupWizardPixelPrebuilt_cd1a \
     StorageManagerGoogle \
     TurboAdapter \
-    WallpaperPickerGoogleRelease \
     YourWidgetsStub
 
 # PrebuiltGmsCore
diff --git a/gms_full_tablet_wifionly.mk b/gms_full_tablet_wifionly.mk
index a23a460..45b4725 100644
--- a/gms_full_tablet_wifionly.mk
+++ b/gms_full_tablet_wifionly.mk
@@ -87,7 +87,6 @@ PRODUCT_PACKAGES += \
     FilesPrebuilt \
     GoogleRestorePrebuilt-v1054408 \
     KidsSupervisionStub \
-    OdadPrebuilt \
     PartnerSetupPrebuilt \
     Phonesky \
     PrebuiltBugle \
@@ -95,14 +94,11 @@ PRODUCT_PACKAGES += \
     PrebuiltPixelCoreServices \
     RelationshipsPrebuilt-301 \
     ScribePrebuilt_v8.7.880674799 \
-    SearchSelectorPrebuilt \
     SettingsIntelligenceGooglePrebuilt \
-    SetupWizardPrebuilt_cd1a \
     TurboPrebuilt \
     Velvet \
     VerifierPrebuiltClassic \
     WallpaperEffect \
-    WeatherPixelPrebuilt_10008663 \
     WellbeingPrebuilt
 
 TARGET_INCLUDE_LIVE_WALLPAPERS ?= true
@@ -124,7 +120,6 @@ PRODUCT_PACKAGES += \
 # system/priv-app
 PRODUCT_PACKAGES += \
     DocumentsUIGoogle \
-    GooglePackageInstaller \
     TagGoogle
 
 # system_ext/app
@@ -136,16 +131,12 @@ PRODUCT_PACKAGES += \
 # system_ext/priv-app
 PRODUCT_PACKAGES += \
     AvatarPickerGoogle \
-    DeviceConnectivityServicePrebuilt_26.01.00 \
     GoogleFeedback \
     GoogleServicesFramework \
     MagicPortraitWallpapers \
     MoseyApp \
-    NexusLauncherRelease \
-    SetupWizardPixelPrebuilt_cd1a \
     StorageManagerGoogle \
-    TurboAdapter \
-    WallpaperPickerGoogleRelease
+    TurboAdapter
 
 # PrebuiltGmsCore
 PRODUCT_PACKAGES += \
diff --git a/gms_mini.mk b/gms_mini.mk
index 28aa0f0..af0c27e 100644
--- a/gms_mini.mk
+++ b/gms_mini.mk
@@ -74,14 +74,12 @@ PRODUCT_PACKAGES += \
     GoogleDialer \
     GoogleRestorePrebuilt-v1054408 \
     KidsSupervisionStub \
-    OdadPrebuilt \
     PartnerSetupPrebuilt \
     Phonesky \
     PrebuiltBugle \
     PrebuiltDeskClockGoogle_76060331 \
     PrebuiltPixelCoreServices \
     SettingsIntelligenceGooglePrebuilt \
-    SetupWizardPrebuilt_cd1a \
     TurboPrebuilt \
     Velvet \
     VerifierPrebuiltClassic \
@@ -119,10 +117,7 @@ PRODUCT_PACKAGES += \
     AvatarPickerGoogle \
     GoogleFeedback \
     GoogleServicesFramework \
-    MoseyApp \
-    NexusLauncherRelease \
-    SetupWizardPixelPrebuilt_cd1a \
-    WallpaperPickerGoogleRelease
+    MoseyApp
 
 # PrebuiltGmsCore
 PRODUCT_PACKAGES += \
diff --git a/gms_pico.mk b/gms_pico.mk
index 7f7e9ce..410bd6a 100644
--- a/gms_pico.mk
+++ b/gms_pico.mk
@@ -55,13 +55,11 @@ PRODUCT_PACKAGES += \
     GoogleDialer \
     GoogleRestorePrebuilt-v1054408 \
     KidsSupervisionStub \
-    OdadPrebuilt \
     PartnerSetupPrebuilt \
     Phonesky \
     PrebuiltDeskClockGoogle_76060331 \
     PrebuiltPixelCoreServices \
     SettingsIntelligenceGooglePrebuilt \
-    SetupWizardPrebuilt_cd1a \
     TurboPrebuilt \
     VelvetGo \
     VerifierPrebuiltClassic \
@@ -75,7 +73,6 @@ PRODUCT_PACKAGES += \
 # system/priv-app
 PRODUCT_PACKAGES += \
     DocumentsUIGoogle \
-    GooglePackageInstaller \
     TagGoogle
 
 # system_ext/app
@@ -89,10 +86,7 @@ PRODUCT_PACKAGES += \
     GoogleFeedback \
     GoogleServicesFramework \
     MoseyApp \
-    NexusLauncherRelease \
-    SetupWizardPixelPrebuilt_cd1a \
-    StorageManagerGoogle \
-    WallpaperPickerGoogleRelease
+    StorageManagerGoogle
 
 # PrebuiltGmsCore
 PRODUCT_PACKAGES += \
